frontend/public/components/_overview.scss

```diff
@@ -26,8 +26,72 @@
 }
 
 .overview {
-  width: 100%;
+  bottom: 0;
   display: flex;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 67px;
```

Comment:
Shoulde use a var.

[Global notifications](https://github.com/openshift/console/blob/master/frontend/public/components/app.jsx#L129) are currently not in use (impersonate user is currently disabled), but this will be problematic if they are re-enabled.

Reply:
I don't think impersonate user is disabled?

Reply:
@spadgett is correct.  Impersonate user is still enabled, it's just a bit tricky to find as not all role binding cogs have the option in the dropdown.  Try "cluster-admin".

<img width="311" alt="screen shot 2018-09-21 at 2 42 00 pm" src="https://user-images.githubusercontent.com/895728/45900042-8c429b00-bdac-11e8-83e5-2f7185e9ad45.PNG">